### ISSUE TITLE
Not applying in MATLAB in Ubuntu or Windows

### ISSUE THREAD

**Issue by kevinjohncutler** [rbreaves/kinto#295]
Kinto is amazing, thank you so much for it. However, it seems like none of the keyboard shortcuts work in Windows 10 or Ubuntu 20.04 _ but only in MATLAB_. Default shortcuts also fail to work, so copy+paste etc. is much more cumbersome haha. Has this sort of thing been reported before with other programs?

**Comment by rbreaves** [comment]
I dog food this on Windows 10 daily & even reinstalled it fresh from the dev branch on Windows 10 & several Linux distros just before merging the latest v1.2 update into master only a few hours ago.

If there’s an issue w/ the hotkeys not working then I definitely want to fix it.

Can you give me any more info?? Did the Windows task tray appear after the install?

Does the Linux sys tray or GUI app indicate that it’s running? Can you enable debug mode from the help menu to monitor if it picks up on you pressing key combos?

**Comment by rbreaves** [comment]
I guess I’m confused too as the title reads as though it’s just MATLAB but then the body of your message makes it sound like it only works in matlab lol. I’ve never used matlab, although I have heard of it. There’s no reason I’m aware of that’d prevent the basic remap from happening.

Only Virtualbox is excluded from remaps but other VM apps ought to be added imo. Also terminals are a modified layout that usually drops the super key in favor of Lctrl, Alt & rctrl (or Ctrl+shift effectively).

**Comment by kevinjohncutler** [comment]
Thanks for the quick response! Sorry for the confusion: Kinto works everywhere except MATLAB. In Ubuntu, I don’t see anything in the system tray / taskbar area. In Windows the AutoHotkey.exe run dialogue always pups up on boot (after about a minute, is there a way to make it start in the background?) and Kinto does indeed appear in the system tray. And actually just now after a system update, it is working in MATLAB! Awesome. Still not working in Ubuntu (again only in this one program, perfect everywhere else). I’ve never seen a GUI for Kinto in Ubuntu though - where should I be able to find it?

**Comment by rbreaves** [comment]
If you type Kinto in the application overview it will appear if you installed the latest v1.2 & yea Ubuntu dropped support for sys trays.. so any DE that supports appindicators will show the sys tray (xfce, Budgie, mate.. probably more).

You can also go straight into editing the config file on the GUI app on Linux.

I’ve not yet updated the sys tray on Windows to match the latest GUI or sys tray app on Linux. To edit the Windows file it’ll be ~/.kinto/kinto.py... not sure why matlab would fail though.. by default all GUI based apps are to get the remap. Terminals getting a slightly different version.

**Comment by rbreaves** [comment]
Also I probably could silence the autohotkey elevation but it’s not a high priority atm. I do silence some elevation activities already, just not that one. May revisit that later.

**Comment by kevinjohncutler** [comment]
Ok cool, just updated to 1.2, and I can open the GUI. As to MATLAB, I found a way to fix it. The default keyboard shortcuts are 'Emacs default set', but I changed it to the 'Windows default set' and now everything is normal. This only affected the MATLAB terminal and script editor (as opposed to the 'live script' editor which is more of a jupyter/mathematica notebook-like editor), but anyway this makes sense - and hopefully anyone else with with problem can find this solution by the many keywords I have included haha.

Thanks again so much for Kinto, it's a life changer.
